src/app/model/event.ts

```diff
@@ -1,6 +1,6 @@
 export class Event {
 
-    id: string;
+    id: number;
```

Comment:
Does this need to be changed to eventId?

Reply:
Disregard! I realized that you changed the name of the field on your search-event-by-keyword pull request :)